glossary.md

```diff
@@ -48,14 +56,72 @@ Coalesce Partition is a way of decreasing the number of partitions in a Hash or
 
 Introduced in TiDB 5.3.0, Continuous Profiling is a way to observe resource overhead at the system call level. With the support of Continuous Profiling, TiDB provides performance insight as clear as directly looking into the database source code, and helps R&D and operation and maintenance personnel to locate the root cause of performance problems using a flame graph. For details, see [TiDB Dashboard Instance Profiling - Continuous Profiling](/dashboard/continuous-profiling.md).
 
+### CTE
+
+A Common Table Expression (CTE) enables you to define a temporary result set that can be referred multiple times within a SQL statement using the [`WITH`](/sql-statements/sql-statement-with.md) clause. For more information, see [Common Table Expression](/develop/dev-guide-use-common-table-expression.md).
+
 ## D
 
+### DDL
+
+Data Definition Language (DDL) statements enables you to create, modify, and drop tables, indexes, columns, and other database objects.
+
+### DM
+
+Data Migration (DM) is a tool for migrating data from MySQL-compatible databases into TiDB. It reads data from an instance of MySQL-compatible database and applies it to a TiDB target instance. For more information, see [DM Overview](/dm/dm-overview.md).
+
+### DML
+
+Data Modification Language (DML) statements enables you to with insert, update, and delete rows in tables.
+
+### DMR
+
+Development Milestone Release (DMR) is a TiDB version that introduces the latest features but does not offer long-term support. For more information, see [TiDB Versioning](/releases/versioning.md).
+
+### DR
+
+Disaster Recovery (DR) includes solutions that can be used to recover data from a disaster in the future. These solutions typically involve backups and standby clusters. For more information, see [Overview of TiDB Disaster Recovery Solutions](dr-solution-introduction).
+
+### DXF
+
+Distributed eXecution Framework (DXF) is the framework used by TiDB for accelerating index creation and data import by distributing tasks over all available resources. For more information, see [DXF Introduction](/tidb-distributed-execution-framework.md).
```

Comment:
Some suggestions to make it clear DXF scope is broader than just index creation and data import tasks, e.g. these are examples, not the only tasks that may be distributed:

Distributed eXecution Framework (DXF) is the framework used by TiDB to distribute tasks across the TiDB cluster. DXF is designed to efficiently use the cluster resources to execute tasks (like index creation or data import) while controlling the resource usage and impact on core business transactions. For more information, see [DXF Introduction](/tidb-distributed-execution-framework.md).